organizations/backends.py

```diff
@@ -1,8 +1,24 @@
+import collections
+import logging
+
 from django.conf import settings
 from django.contrib.auth.backends import ModelBackend
 from django.core.exceptions import PermissionDenied
-from openedx.core.djangoapps.theming.helpers import get_current_site
-from openedx.core.djangoapps.site_configuration.helpers import is_site_configuration_enabled, get_value
+
+try:
```

Comment:
@bryanlandia First, thanks for adding test coverage for backends, much needed addition!

However I strongly urge against embedding test code in production code.  I understand the challenge of mocking imports to modules that don't exist for the test environment and this was the solution you found. However, I see it as an avoidable anti-pattern. 

I had to address the same problem in `figures.compat` in order to test agains non-existing platform code.  Here is how I did it:

The basic approach is to patch sys.modules for your test using `mock.patch.dict`. Here is a convenience function I created:

https://github.com/appsembler/figures/blob/master/tests/test_compat.py#L33-L60

Then in the test, you can simply use `patch_module` as following: 

https://github.com/appsembler/figures/blob/master/tests/test_compat.py#L130-L161

@OmarIthawi What are your thoughts here?

Reply:
Regarding mocking, I think Figures code provides a lot of quality examples which I've reused in the [Course Access Groups repo](https://github.com/appsembler/course-access-groups).